official/guides/child_account_update/export.py

```diff
@@ -0,0 +1,173 @@
+# This script will export USPS credentials for all child accounts to a CSV file
+
+# NOTE: Due to the nature of the easypost API library,
+# this script will run synchronously and therefore may take a while to complete if you have a lot of child accounts
+
+import argparse
+import csv
+from typing import (
+    Any,
+    Dict,
+    List,
+    Union,
+)
+
+import easypost
+
+
+OUTPUT_FILE_NAME = "child_accounts.csv"
+
+parser = argparse.ArgumentParser(description="Export all child account USPS production credentials")
+parser.add_argument("-k", "--key", required=True, help="Parent production API key")
+
+
+def authenticate(key: str):
+    """
+    Authenticate with the given API key
+
+    :param key: API key to authenticate with
+    :type key: str
+    :raises ValueError: If the key is invalid
+    :return: None
+    """
+    if not key:
+        raise ValueError("API key not provided")
+    easypost.api_key = key
+
+
+def get_production_key(keys: List[Dict[str, Any]]) -> Union[str, None]:
+    """
+    Get the production key from a list of keys
+
+    :param keys: List of keys
+    :type keys: List
+    :return: Production key if found, None otherwise
+    :rtype: str | None
+    """
+    for key in keys:
+        if key["mode"] == "production":
+            return key["key"]
+    return None
+
+
+def remap_credentials(credentials: Dict[str, Any]) -> Dict[str, Any]:
+    """
+    Remap a credentials dictionary to only necessary key-value pairs
+
+    :param credentials: Credentials dictionary of a USPS account
+    :type credentials: Dict[str, Any]
+    :return: Remapped credentials dictionary
+    :rtype: Dict[str, Any]
+    """
+    remapped = {}
+    if not credentials:
+        return remapped
+    for key, value in credentials.items():
```

Comment:
At the least, this needs to be `to_dict` and we should probably remove this entirely if not needed.

```suggestion
    for key, value in credentials.to_dict().items():
```

Reply:
Resolved